plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveMetadataListing.java

```diff
@@ -139,6 +139,7 @@ public void testViewListing()
                 FAILING_STORAGE_DESCRIPTOR_VIEW.getSchemaTableName().getTableName());
         assertQueryReturnsEmptyResult(withSchemaAndFailingSDViewFilter);
 
+        // TODO This should be improved to not return empty results https://github.com/trinodb/trino/issues/21990
```

Comment:
( https://github.com/trinodb/trino/issues/21990 )

Reply:
- what else would you expect?
- consider linking https://github.com/trinodb/trino/issues/6551